projects/packages/my-jetpack/_inc/components/product-card/style.module.scss

```diff
@@ -67,6 +67,8 @@
 .status {
   margin-left: var(--product-card-margin-base);
   white-space: nowrap;
+  height: 36px;
+  line-height: 36px;
```

Comment:
Use the var here: var(--product-card-actions-size)

Reply:
addressed here ec654f582eed052d85cf8a8a886d83a87c6d3803